packages/node_modules/pouchdb-adapter-indexeddb/src/bulkDocs.js

```diff
@@ -314,6 +314,17 @@ export default function (api, req, opts, metadata, dbOpts, idbChanges, callback)
     };
   }
 
+  function refreshMetadataSeq() {
+    return new Promise(function (resolve) {
+      txn.objectStore(DOC_STORE).index('seq').openCursor(null, 'prev').onsuccess = function (evt) {
```

Comment:
Would this have the same effect as:
```js
metaStore.get(META_STORE).onsuccess = function (e) {
  metadata.seq = e.target.result.seq;
  ...
}
```
?

Reply:
I will have to spend some time analyzing it. It's been too long for me to remember the technical details.

Reply:
Yes it does. It seems to be a better solution so I've merged and updated the code to match this technique.